packages/components/src/components/icon/icon.stories.ts

```diff
@@ -8,26 +8,123 @@ const { overrideArgs } = storybookHelpers('sd-icon');
 const { generateTemplate } = storybookTemplate('sd-icon');
 
 /**
- * ## Disclaimer
- * The `sd-icon` component does not contain any icons itself. It only provides a technical way to implement and use icons from a specfic library (see below).
- * All icons displayed here, are provided by the brand departement and requests towards the icons itself need to be addressed accordingly.
- * In order not to depend on any external library, the SDS-Library includes a small stack of "built-in" icons (e.g. close, chevron etc), which are maintained by the SDS team.
+ * **Used to display icons.**
  *
- * ## Colors
- * Per default icons inherit their color from the current text color. Thus, you can set the color property on the <sd-icon> element or an ancestor to change the color.
+ * Icons can be used to indicate an action or to represent content.
  *
- * ## Sizes
- * Icons are sized relative to the current font size. To change their size, set the font-size property on the icon itself or on a parent element as shown below.
+ */
+
+export default {
+  title: 'Components/sd-icon',
+  component: 'sd-icon',
+  args: overrideArgs([{ name: 'name', type: 'attribute', value: 'union-investment/content/image' }], args),
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  }
+};
+
+/**
+ * Use the `name` attribute to change the icon. Checkout these lists of Union Investment's icons for available names:
+ * - [Content Icons](?path=/story/components-sd-icon-default--library-default-content)
+ * - [System Icons](?path=/story/components-sd-icon-default--library-default-system)
```

Comment:
do you see another way for the naming? we show the example to register the GR icons down in the story, where we also use the term `system` as the GR library is called system. 

I see a risk of confusion here as it is quite complex anyways and it will be even more complex as soon as we provide the GR libraries in our DS.

Reply:
actually thats what I meant. like the idea of using `internal` instead.